MT-CTF/capturetheflag
# Add CTF Respawn Delay

Closes #646

## How to test
* Kill and be killed with multiple clients
* Make sure everything that normally happens after death/respawn will still happen
* See how the 5 second delay feels. Does it need tweaking?
